lib/pageflow/configuration.rb

```diff
@@ -444,6 +450,14 @@ def paperclip_filesystem_root=(_val)
       ActiveSupport::Deprecation.warn('Pageflow::Configuration#paperclip_filesystem_root is deprecated.', caller)
     end
 
+    def for_entry_type(type, &_block)
+      type_name = type.name
+      config = entry_types.find_by_name!(type_name).configuration.new(self)
+      entry_type_configs[type_name] = config
+
+      yield(config)
+    end
```

Comment:
Calling `for_entry_type` a second time overwrites a previously created config. This case should be covered by a spec.